### DISCUSSION TITLE
error: 'sensor_t {aka struct _sensor}' has no member named 'set_xclk'

### DISCUSSION THREAD

**Discussion by commentsonly** [easytarget/esp32-cam-webserver#245] [Common Issues] [answered]
no idea about that ...... happy to press compile and far from being a software engineer.
If only the readme would have explained which step I might have missed
.

**Comment by easytarget** [comment, marked as answer]
Please use the boards manager to upgrade to ESP32 version 2.0.2 (the latest).
- You have a really old version installed, 1.0.4 or earlier.

I'm not sure which readme you were following.. but the one I wrote says to install the latest esp Arduino core via the boards manager. Make sure you have correctly followed the [esp32 Arduino Core](https://docs.espressif.com/projects/arduino-esp32/en/latest/installing.html) setup instructions as explained in the [setup](https://github.com/easytarget/esp32-cam-webserver#setup) section of the readme.

**Comment by commentsonly** [comment]
Dammit ... yes I was fiddeling with an example where I had to switch to and older version and forgot about reverting this. Shame on me. Thanks for the hint.

**Reply by easytarget** [reply]
No worries, at lease I know what that error means now too, i'll leave this in the discussion so it if findable by others.
